.travis.yml

```diff
@@ -51,7 +51,7 @@ before_script:
 script:
   - java -version
   - git clone https://github.com/scala-native/sbt-cross.git
-  - pushd sbt-cross
+  - pushd sbt-crossproject
   - sbt publishLocal
   - popd
```

Comment:
Why is this still necessary? Shouldn't we rely on published version of `sbt-crossproject` now?